linkis-public-enhancements/linkis-configuration/src/main/java/org/apache/linkis/configuration/restful/api/ConfigurationRestfulApi.java

```diff
@@ -436,6 +437,10 @@ public Message saveKeyValue(HttpServletRequest req, @RequestBody Map<String, Obj
     if (StringUtils.isBlank(configKey) || StringUtils.isBlank(value)) {
       return Message.error("key or value cannot be empty");
     }
+    if (RMConfiguration.USER_AVAILABLE_MEMORY().key().equals(configKey)
+        && !ConfigurationConfiguration.confRegexPattern.matcher(value).matches()) {
+      return Message.error("The value " + value + " should be in g|G");
+    }
```

Comment:
can refer to  
org.apache.linkis.configuration.service.ConfigurationService#paramCheck

Reply:
done